src/data-mgt/python/devices-tranformation/transformation.py

```diff
@@ -134,7 +134,7 @@ def map_sites_to_tahmo_station(self):
 
```

Comment:
So this looks like a file changed from another microservice, intentional? @NoahNsimbe

Reply:
yes @Baalmart. that is check box 5 in the PR description. `Fixes a typo on the device/site utility functions`